src/main/java/com/litequizapp/server/CategoryRestController.java

```diff
@@ -20,34 +24,37 @@ public CategoryRestController(CategoryRestService restService) {
     this.restService = restService;
   }
 
-  @GetMapping("/all")
-  public ResponseEntity getAllCategories() {
+  @GetMapping
+  public ResponseEntity<List<String>> getAllCategories() {
     return ResponseEntity.ok(restService.getAllCategory());
   }
 
-  @GetMapping("/id/{id}")
-  public ResponseEntity getCategoryByIdRequest(@PathVariable String id) {
-    long Id = Long.parseLong(id);
-    return ResponseEntity.ok(restService.getCategoryById(Id));
+  @GetMapping(value = "/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<String> getCategoryByIdRequest(@PathVariable String id) {
+    long categoryId = Long.parseLong(id);
+    return ResponseEntity.ok(restService.getCategoryById(categoryId));
   }
 
-  @GetMapping("/title/{title}")
-  public ResponseEntity getCategoryByTitleRequest(@PathVariable String title) {
-    return ResponseEntity.ok(restService.getCategoryByTitle(title));
 
-  }
-
-  @PostMapping("/create")
+  @PutMapping
   public void createCategoryRequest(@RequestBody String title) {
     restService.createCategory(title);
 
   }
 
-  @PostMapping("/delete")
+  @PostMapping(value = "/{id}")
+  public void updateCategoryRequest(@PathVariable String id, @RequestBody String title) {
+    long categoryId = Long.parseLong(id);
+    restService.updateCategory(categoryId, title);
+
+  }
+
+  @DeleteMapping
   public void deleteCategoryByIdRequest(@RequestBody String id) {
```

Comment:
It's not intuitive that we had all previous operation by id with id as path variable, and delete with id as body.